code/modules/materials/material_sheets.dm

```diff
@@ -403,7 +403,7 @@
 	singular_name = "biomatter sheet"
 	icon_state = "sheet-biomatter"
 	default_type = MATERIAL_BIOMATTER
-	price_tag = 10
+	price_tag = 5
```

Comment:
```suggestion
	price_tag = 7
```
maybe take it slow

Reply:
This is going with a PR that removes the bottleneck. One person should be able to make 3x as much biomatter by themselves in an easier manor. A medium pouch right now costs 45 in biomatter, the price is to bring it in line with the actual value of the items it's in and the tank buff is to make it so that NT makes the same if not more money per effort. It just needs more plants but let's be honest you can max out production with 4 grass plants unmodded right now. Get some glowshroom grass and you'll max out this one with like 10 plants planted in the reactor room